src/vl-map-layer-circle-style.js

```diff
@@ -73,7 +73,7 @@ export class VlMapLayerCircleStyle extends VlMapLayerStyle {
     return (feature, resolution) => {
       const features = feature && feature.get ? (feature.get('features') || []) : [];
       const size = features.length || 1;
-      const clusterMultiplier = size == 1 ? 1 : Math.max(3, size.toString().length);
+      const clusterMultiplier = size == 1 ? 1 : Math.max(1.5, size.toString().length);
```

Comment:
Ik volg niet goed waarom de lengte van de features.size.toString() bepalend is voor een cluster.

Reply:
Er moet meer plaats voorzien worden in de cluster voor een cijfer met meerdere cijfers, 1 vs 10 vs 100.

Reply:
Ok, ik had niet genoeg rondgekeken in de context van de file, in dat geval. 
Dat gezegd zijnde, als je het capped op 1,5 (of eender welk getal) dan ga je toch in bepaalde situaties (als er echt grote clusters op de kaart staan) toch snel tegen een "probleem" aanstoten?

Reply:
Hij neemt het maximum van 1,5 of het aantal cijfers. 3 werd even gebruikt in kader van de huisstijl aanpassingen maar had geen gewenst resultaat in combinatie met clusters. Ik heb 1,5 gereverted omdat deze waarde reeds jaren gebruikt werd.

Reply:
Ah sorry, ik was het aan het lezen als "1.5 als bovengrens" ipv. "1.5 als ondergrens". Denkfout van mij :)